sig-contributor-experience/README.md

```diff
@@ -13,7 +13,7 @@ Developing and sustaining a healthy community of contributors is critical to sca
 The [charter](charter.md) defines the scope and governance of the Contributor Experience Special Interest Group.
 
 ## Meetings
-* Regular SIG Meeting: [Wednesdays at 9:30 PT (Pacific Time)](https://zoom.us/j/397264241?pwd=bHNnZVArNFdPaWVJMmttdko0Sktudz09) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:30&tz=PT%20%28Pacific%20Time%29).
+* Regular SIG Meeting: [Wednesdays at 9:00 PT (Pacific Time)](https://zoom.us/j/397264241?pwd=bHNnZVArNFdPaWVJMmttdko0Sktudz09) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:30&tz=PT%20%28Pacific%20Time%29).
```

Comment:
@spzala The source of truth is in sigs.yaml so that'll need to get updated first. Running `make` will autogenerate these files. :)

I think it'd be great if we could include the slack meetings too in case folks want to participate there :+1:

Reply:
:) ah, thanks @nikhita that helps! I was late this week for meeting and thought may be others can run into it if not watching slack so quickly created this PR while in the meeting :) Yes, agree on including slack meetings here. Let me update sigs.yaml, but I guess slack time and other info (I believe we are creating a new channel for meeting) is not yet finalized? So may need to wait on that.